### DISCUSSION TITLE
Lesson 5 Issue with running 'brownie run .\scripts\deploy.py'

### DISCUSSION THREAD

**Discussion by heider89** [smartcontractkit/full-blockchain-solidity-course-py#995] [Q&A] [answered]
Hi, I refer to the Youtube video at the 4:34:10 mark. I'm facing the following error when running deploy.py file
![error](https://user-images.githubusercontent.com/85205030/153125435-f0234127-31d6-44bf-9f7b-67f9864ce17a.png)

I notice that a similar question was raised before (https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/39) but the solution did not work on my end. Kindly advice.
.

**Comment by heider89** [comment, marked as answer]
Okay apparently, it is solved by changing (or creating a new) the directory. 
1. Old: C:\Users\heide\ .vscode\demos\brownie_simple_storage> 
2. New: C:\Solidity\brownie_simple_storage>

Would be interested to know why this is the case.

**Comment by n4n0b1t3** [comment]
I had a similar experience two days ago. Before restarting the whole thing in a new directory you can try the following things:
1. Just try a few times to run the script. I had issues with the first run but none with the second run. It seems to me that there is a lot of communication between brownie and ganache happening and brownie does not seem to be rock solid (from my few days of experience).
2. I would close the terminal, open a new one and try again.
3. A mysterious compiler issue was solved by removing all compiler artifacts within the build directory and subdirectories and trying again.

Did you install the new ganache or the deprecated ganache-cli? If so, please have a look at this post https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/971

**Comment by P1X37s** [comment]
Just a note to say I had the same issue. Resolved by re-installing ganache globally from the cmd line

npm install ganache --global
